include/clang/Driver/Options.td

```diff
@@ -2625,6 +2625,9 @@ def : Flag<["-"], "integrated-as">, Alias<fintegrated_as>, Flags<[DriverOption]>
 def : Flag<["-"], "no-integrated-as">, Alias<fno_integrated_as>,
       Flags<[CC1Option, DriverOption]>;
 
+def fdump_extracted_comparison_facts : Flag<["-"], "fdump_extracted_comparison_facts">, Group<f_Group>, Flags<[CC1Option]>,
```

Comment:
The actual flag should be dash (-) separated, not underscore separated.
`"fdump_extracted_comparison_facts" ==> "fdump-extracted-comparison-facts"`

Reply:
Done.